packages/apsara-ui/src/TableV2/VirtualisedTable.tsx

```diff
@@ -37,7 +48,7 @@ function VirtualisedTable({
     loading = false,
 }: ITableProps) {
     useEffect(() => {
-        if (!dataFetchFunction && items?.length) {
+        if (!dataFetchFunction) {
```

Comment:
Let's add a default value for `items` prop so that there is no crash incase it's undefined somehow.